src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.scss

```diff
@@ -37,6 +37,18 @@ p {
   .info {
     color: bd.$info;
   }
+
+  .debug {
+    color: bd.$gray-700;
+  }
+}
+
+::ng-deep timepicker table tbody tr td {
+  input.bs-timepicker-field {
+    width: 3.5rem !important;
+    font-size: 1rem;
+    padding: 4px 6px;
+  }
```

Comment:
We no longer use this timepicker.
Please remove this or adapt it if still necessary.